Tests/RxCocoa/unrwapTests+SharedSequence.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  UnwrapTests.swift
+//  RxSwiftExt
+//
+//  Created by Hugo Saynac on 05/10/2018.
+//  Copyright © 2018 RxSwift Community. All rights reserved.
+//
+
+import XCTest
+
+import RxSwift
+import RxSwiftExt
+import RxTest
+
+class UnwrapSharedStrategyTests: XCTestCase {
+    let numbers: [Int?] = [1, nil, Int?(3), 4]
+    fileprivate var observer: TestableObserver<Int>!
+    fileprivate let numbersSubject = BehaviorSubject<Int?>(value: nil)
+
+    override func setUp() {
+        super.setUp()
+
+        let scheduler = TestScheduler(initialClock: 0)
+        observer = scheduler.createObserver(Int.self)
+
+        _ =  numbersSubject
+                .asDriver(onErrorJustReturn: nil)
+                .unwrap()
+                .asObservable()
+                .subscribe(observer)
+
+        _ = Observable.from(numbers)
+            .bind(to: numbersSubject)
+
+        scheduler.start()
+    }
+
+    func testUnwrapFilterNil() {
+        //test results count
+        XCTAssertEqual(
+            observer.events.count,
+            numbers.count /* -1 the number of nil elements, +1 completed event */
```

Comment:
Not sure this comment makes sense in this context - could be a copy-paste left over from a different test?

Reply:
I think it's fine removing this comment as there's no -1/+1 here so its just adding confusion.

Reply:
Removed ! 👍